Doc/library/tracemalloc.rst

```diff
@@ -289,6 +289,26 @@ Functions
    :mod:`tracemalloc` module as a tuple: ``(current: int, peak: int)``.
 
 
+.. function:: reset_peak()
+
+   Set the peak size of memory blocks traced by the :mod:`tracemalloc` module
+   to the current size.
+
+   The peak memory usage of a specific section of code can be determined as
+   follows::
```

Comment:
I'm not sure that this explanation belongs to the "reference API documentation". Maybe add an example showing how to get the new peak in the following section?
https://docs.python.org/dev/library/tracemalloc.html#examples

Reply:
Done!

Reply:
Ah, after doing this, I realised that it might be better to be a more general example. I've changed it to "Record the current and peak size of all traced memory blocks", that includes more discussion of `get_traced_memory` itself.

Reply:
It's now better with an example! Thanks.